core/jazz_metrics/config/dev-config.json

```diff
@@ -1,6 +1,7 @@
 {
   "SERVICE_API_URL": "https://{conf-apikey}.execute-api.{conf-region}.amazonaws.com",
   "ASSETS_URL": "/dev/jazz/assets",
+  "SERVICE_URL": "/dev/jazz/services",
   "TOKEN_URL": "/dev/jazz/login",
```

Comment:
"SERVICE_API_URL": "https://{conf-apikey}.execute-api.{conf-region}.amazonaws.com/dev",

And remove the dev from the rest of the values

Reply:
Done also updated for stg and prod config too.